src/JustEat.StatsD/EndpointLookups/DnsEndpointProvider.cs

```diff
@@ -10,7 +10,8 @@ public class DnsEndpointProvider : IDnsEndpointMapper
     {
         public IPEndPoint GetIPEndPoint(string hostName, int port)
         {
-            return new IPEndPoint(Dns.GetHostAddresses(hostName)[0], port);
+            var endpoints = Dns.GetHostAddressesAsync(hostName).Result;
```

Comment:
Are we ok with the use of `.Result`, or do we have to make the whole thing async?